drivers/wifi/Kconfig

```diff
@@ -29,6 +29,11 @@ config WIFI_OFFLOAD
 	help
 	  Enable support for Full-MAC Wi-Fi devices.
 
+config WIFI_USE_NATIVE_NETWORKING
+	bool "Support Wi-Fi device drivers using native network stack"
```

Comment:
Wouldn't make it sense to make this symbol hidden to the user, and only selected from drivers when required.

Reply:
Makes sense to me!

Reply:
I agree. Updated.